pkgdb/tests/lock-flake-installable.cc

```diff
@@ -0,0 +1,372 @@
+/* ========================================================================== *
+ *
+ *  @file lock-flake-installable.cc
+ *
+ *  @brief Tests for `lock-flake-installable` functionality.
+ *
+ * -------------------------------------------------------------------------- */
+
+#include "flox/lock-flake-installable.hh"
+#include "flox/core/util.hh"
+#include "test.hh"
+#include <fstream>
+#include <nix/flake/flake.hh>
+
+/* -------------------------------------------------------------------------- */
+
+
+/* -------------------------------------------------------------------------- */
+
+std::string
+unsupportedPackage( const std::string & system )
+{
+  if ( system == "aarch64-darwin" ) { return "glibc"; }
+  else if ( system == "x86_64-darwin" ) { return "glibc"; }
+  else if ( system == "aarch64-linux" ) { return "spacebar"; }
+  else if ( system == "x86_64-linux" ) { return "spacebar"; }
+  else
+    {
+      // Should be unreachable
+      return "wat?";
+    }
+}
+
```

Comment:
nonblocking: is this used?
```suggestion
```

Reply:
no, i left it in originally in case i need to check that the local system doesnt matter to eval.
We have that in another test now, will remove